tests/coalaHTMLTest.py

```diff
@@ -40,10 +54,14 @@ def test_output_file(self):
                           "-c", os.devnull,
                           "-b", "LineCountBear",
                           "-f", re.escape(filename),
-                          "--noupdate",
-                          "--nolaunch")
+                          "--noupdate")
 
             with open(self.result_file, 'r') as fp:
                 noupdate_file = fp.read()
 
         self.assertEqual(update_file, noupdate_file)
+        # We `chdir` before spawning the server
+        shutil.rmtree('../coalahtmlapp', ignore_errors=True)
```

Comment:
No. I do `chdir` just before spawning the server in `coala_html.py` at [L76](https://github.com/coala-analyzer/coala-html/blob/tushar-rishav/coalahtml/coala_html.py#L76) :sweat_smile: 
Um we needn't put all cleanup into tearDown because the `rmtree` is required only when coala_html is run (as it creates necessary directories). Other test suites might not need to run coala_html. So In such case the clean up isn't required. For now the clean up is required only for this particular suite. Maybe we put all clean ups to tearDown when they are needed in other suites and leave for now?